.env

```diff
@@ -2,22 +2,22 @@
 VERBOSE_HARNESS=0
 
 # Used to determine sandbox build type:
-TYPE="channel" # OR "source"
+TYPE="source" # OR "source"
 
 # Used when TYPE==channel:
-ALGOD_CHANNEL="nightly"
+ALGOD_CHANNEL=""
 
 # Used when TYPE==source:
 ALGOD_URL="https://github.com/algorand/go-algorand"
-ALGOD_BRANCH="master"
+ALGOD_BRANCH="feature/avm-box"
 ALGOD_SHA=""
 
 # Used regardless of TYPE:
 NETWORK_TEMPLATE="images/algod/DevModeNetwork.json"  # refers to the ./images directory in the sandbox repo
 NETWORK_NUM_ROUNDS=30000
 INDEXER_URL="https://github.com/algorand/indexer"
 NODE_ARCHIVAL="False"
-INDEXER_BRANCH="develop"
+INDEXER_BRANCH="master"
```

Comment:
```suggestion
INDEXER_BRANCH="tzaffi/box-search"
```

Reply:
I think we are going to start indexer integration test, so I will use this code suggestion, and add a suggestion once we are reverting back to `develop`.